### DISCUSSION TITLE
Quais cursos realizar ?

### DISCUSSION THREAD

**Discussion by TamLuzs** [frontendbr/forum#1442] [Discussão]
Olá,
Atualmente estou interessada em realizar o curso de Design Digital(Anhembi Morumbi) ou Sistema Para Internet(FIAP), enfim gostaria obter máximo de informação antes de iniciar algum desses cursos. 
**Meu foco atual é:** UI Design / Desenvolvedor Front-end / Desenvolvedor JS

Tenho noções / breve conhecimento em algumas linguagens de programação (C#, C, HTML (Linguagem de Hipertexto), CSS (Linguagem de Estilo) e SQL ).

Ao pesquisar cursos online, encontrei os seguintes cursos abaixo.
- https://www.udemy.com/curso-completo-do-desenvolvedor-web/ (Finalizando -> Introdutório)
- https://www.devmedia.com.br/front-end/ (Ponto positivo: Parece ter diversos desafios)
- https://www.treinaweb.com.br/formacao/desenvolvedor-front-end-junior (Ser separado por Niveis (JR, PL e SR) )
- https://www.alura.com.br/formacao-front-end (Ter diversos focos de formação de front-end)
- https://www.udemy.com/course/curso-web/   (Apresenta diversos pontos intermediário)

**A questão é qual o curso mais indicado para dar continuidade ou plataforma ?**

Obrigada para todos que leram e todos os que comentaram.

**Comment by onlurking** [comment]
Eu tenho as seguintes recomendações pra Front-end e Design:
Começar pelo tutorial do Interneting is hard: https://internetingishard.com/html-and-css, o conteúdo está em inglês, entretanto é o melhor tutorial de frontend que conheço de longe, recomendo ler por completo e fazer todos os exemplos dele.

Depois solidificar os conhecimentos em design, html e css com os cursos da Origamid: 
https://www.origamid.com, o instrutor é absurdo quando se trata de layout, atualmente estou fazendo o nanodegree de frontend da Udacity (que teve o material desenvolvido em parceria com a Google) e na real, o material da Origamid é muito superior, sem brincadeira. Se você mandar um e-mail pro Andre da Origamid ele faz o acesso vitalício pelo valor do plano anual.

Se quiser ter uma noção da qualidade do material do cara:
https://origamid.com/projetos/flexbox-guia-completo
https://origamid.com/projetos/css-grid-layout-guia-completo
https://origamid.com/projetos/wildbeast

Pra referências eu sempre uso os site da Mozilla (MDN), https://cssreference.io e https://htmlreference.io que é do criador do Bulma.

Entretanto o mais importante de tudo é a prática, eu recomendo pegar uns layouts por aí e fazer na unha pra praticar, um lugar bom pra pegar inspiração é o site da Pixelarity: https://pixelarity.com

**Reply by brizareis** [reply]
Muito obrigada pelo seu post @onlurking fantástico!

**Reply by DanielFernandoDias** [reply]
Teria o e-mail do André da Origamid?

**Comment by TamLuzs** [comment]
Diogo, Boa Tarde!

Obrigada pelas recomendações.

Realmente, os cursos Origamid parecem muito bom. 

Utilizo o material da Mozilla para tirar dúvidas neste momento.

**Comment by merciakeila** [comment]
> Eu tenho as seguintes recomendações pra Front-end e Design:
> Começar pelo tutorial do Interneting is hard: https://internetingishard.com/html-and-css, o conteúdo está em inglês, entretanto é o melhor tutorial de frontend que conheço de longe, recomendo ler por completo e fazer todos os exemplos dele.
> 
> Depois solidificar os conhecimentos em design, html e css com os cursos da Origamid:
> https://www.origamid.com, o instrutor é absurdo quando se trata de layout, atualmente estou fazendo o nanodegree de frontend da Udacity (que teve o material desenvolvido em parceria com a Google) e na real, o material da Origamid é muito superior, sem brincadeira. Se você mandar um e-mail pro Andre da Origamid ele faz o acesso vitalício pelo valor do plano anual.
> 
> Se quiser ter uma noção da qualidade do material do cara:
> https://origamid.com/projetos/flexbox-guia-completo
> https://origamid.com/projetos/css-grid-layout-guia-completo
> https://origamid.com/projetos/wildbeast
> 
> Pra referências eu sempre uso os site da Mozilla (MDN), https://cssreference.io e https://htmlreference.io que é do criador do Bulma.
> 
> Entretanto o mais importante de tudo é a prática, eu recomendo pegar uns layouts por aí e fazer na unha pra praticar, um lugar bom pra pegar inspiração é o site da Pixelarity: https://pixelarity.com

Manoo, MUITO OBRIGADA!! Precisava de tudoo isso!

**Comment by mdandrade** [comment]
Confirmo a qualidade da Origamid, principalmente se você estiver muito próximo de design ou for freela.

**Comment by DaianePS13** [comment]
Também reafirmo a qualidade dos cursos da Origamid, adquiri faz uns 6 meses e no quesito Design o professor é excepcional!! Para praticar o que aprende, recomendo também o https://www.freecodecamp.org/, lá ensina várias liguagens que são separadas por trilha e no fim de cada trilha tem um projeto real que você entrega, ele será avaliado, se não me engano dão certificado, é que estou fazendo ele, ainda não cheguei na parte do projeto. 
Sucesso para ti!

**Comment by wilsonneto-swe** [comment]
Outro que me ajudou demais e recomendo bastante são os cursos do devPleno:
http://bit.ly/reactjs-dev-pleno

Focado em react e react native, foi o primeiro que fiz e me deu um ótimo embsamento.

**Comment by vilsonsampaio** [comment]
> Eu tenho as seguintes recomendações pra Front-end e Design:
> Começar pelo tutorial do Interneting is hard: https://internetingishard.com/html-and-css, o conteúdo está em inglês, entretanto é o melhor tutorial de frontend que conheço de longe, recomendo ler por completo e fazer todos os exemplos dele.
> 
> Depois solidificar os conhecimentos em design, html e css com os cursos da Origamid:
> https://www.origamid.com, o instrutor é absurdo quando se trata de layout, atualmente estou fazendo o nanodegree de frontend da Udacity (que teve o material desenvolvido em parceria com a Google) e na real, o material da Origamid é muito superior, sem brincadeira. Se você mandar um e-mail pro Andre da Origamid ele faz o acesso vitalício pelo valor do plano anual.
> 
> Se quiser ter uma noção da qualidade do material do cara:
> https://origamid.com/projetos/flexbox-guia-completo
> https://origamid.com/projetos/css-grid-layout-guia-completo
> https://origamid.com/projetos/wildbeast
> 
> Pra referências eu sempre uso os site da Mozilla (MDN), https://cssreference.io e https://htmlreference.io que é do criador do Bulma.
> 
> Entretanto o mais importante de tudo é a prática, eu recomendo pegar uns layouts por aí e fazer na unha pra praticar, um lugar bom pra pegar inspiração é o site da Pixelarity: https://pixelarity.com

Mano, muito obrigado!!! Não faz ideia como me ajudou, estava muito perdido!

**Comment by alexlopesbr** [comment]
Concordando com os colegas, a qualidade da Origamid é absurda e eu me sinto totalmente satisfeito com a qualidade dos cursos

**Comment by ghost** [comment]
Oi, eu vi essa postagem e queria saber como você tá atualmente na área de UI Design. Eu to fazendo uns cursos lá na origamid (to gostando demais) mas ando meio perdido ainda.. Como foi (está sendo) seu caminho na área?

**Comment by nazarepiedady** [comment]
Para qualquer curso que se for realizar é sempre altamente recomendável ter bibliografia do assunto, ter um coleção vasta e organizada de livros que sanem suas dúvidas sobre o passado, presente e futuro de algo.
